mindee/input/local_response.py

```diff
@@ -0,0 +1,90 @@
+import hashlib
+import hmac
+import io
+import json
+from pathlib import Path
+from typing import Any, BinaryIO, Dict, Union
+
+from mindee.error import MindeeError
+
+
+class LocalResponse:
+    """Local response loaded from a file."""
+
+    _file: BinaryIO
+    """File object of the local response."""
+
+    def __init__(self, input_file: Union[BinaryIO, str, Path, bytes]):
+        if isinstance(input_file, BinaryIO):
+            self._file = input_file
+            self._file.seek(0)
+        elif isinstance(input_file, (str, Path)):
+            with open(input_file, "r", encoding="utf-8") as file:
+                self._file = io.BytesIO(
+                    file.read().replace("\r", "").replace("\n", "").encode()
```

Comment:
I'd be curious to see if it still works without this. You are already stripping whitespace in the test.

The danger of doing this here is that if the server changes to a pretty-printed response, the function will no longer work.

Reply:
I implemented this specifically because it did _not_ work with the provided signature otherwise. I'm all for not doing that, but that means changing it back in java & using a new value.

Reply:
nevermind, my bad